Prometheus example with ddsketch aggregator panics

In exmplae/prometheus/main.go, modify aggregator selector from Exact to Sketch

```
- 	selector := simple.NewWithExactMeasure()
+ 	selector := simple.NewWithSketchMeasure(ddsketch.NewDefaultConfig())
```


Then the whole example will panic with following stacktrace:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x13b8326]

goroutine 9 [running]:
github.com/DataDog/sketches-go/ddsketch.(*DDSketch).Merge(0x0, 0xc00009d740)
        /Users/zhangletian/Dev/go/pkg/mod/github.com/!data!dog/sketches-go@v0.0.0-20190923095040-43f19ad77ff7/ddsketch/ddsketch.go:92 +0x36
go.opentelemetry.io/otel/sdk/metric/aggregator/ddsketch.(*Aggregator).Merge(0xc0001002a0, 0x1545e40, 0xc00009d770, 0xc00007a780, 0xc000126000, 0x2)
        /Users/zhangletian/Dev/go/src/go.opentelemetry.io/otel/sdk/metric/aggregator/ddsketch/ddsketch.go:131 +0x122
go.opentelemetry.io/otel/sdk/metric/batcher/defaultkeys.(*Batcher).Process(0xc0000305c0, 0x1547240, 0xc000028078, 0xc00007a780, 0xc00009d620, 0x1, 0x1, 0xc000028530, 0x10, 0x153ea80, ...)
        /Users/zhangletian/Dev/go/src/go.opentelemetry.io/otel/sdk/metric/batcher/defaultkeys/defaultkeys.go:131 +0x70c
go.opentelemetry.io/otel/sdk/metric.(*SDK).checkpoint(0xc0000af180, 0x1547240, 0xc000028078, 0xc0000ac6e0, 0x1)
        /Users/zhangletian/Dev/go/src/go.opentelemetry.io/otel/sdk/metric/sdk.go:429 +0x257
go.opentelemetry.io/otel/sdk/metric.(*SDK).Collect(0xc0000af180, 0x1547240, 0xc000028078, 0x0)
        /Users/zhangletian/Dev/go/src/go.opentelemetry.io/otel/sdk/metric/sdk.go:393 +0x194
go.opentelemetry.io/otel/sdk/metric/controller/push.(*Controller).collect(0xc0000f0100, 0x1547240, 0xc000028078)
        /Users/zhangletian/Dev/go/src/go.opentelemetry.io/otel/sdk/metric/controller/push/push.go:181 +0x91
go.opentelemetry.io/otel/sdk/metric/controller/push.(*Controller).tick(0xc0000f0100)
        /Users/zhangletian/Dev/go/src/go.opentelemetry.io/otel/sdk/metric/controller/push/push.go:164 +0x4f
go.opentelemetry.io/otel/sdk/metric/controller/push.(*Controller).run(0xc0000f0100, 0xc000092120)
        /Users/zhangletian/Dev/go/src/go.opentelemetry.io/otel/sdk/metric/controller/push/push.go:155 +0x42
created by go.opentelemetry.io/otel/sdk/metric/controller/push.(*Controller).Start
        /Users/zhangletian/Dev/go/src/go.opentelemetry.io/otel/sdk/metric/controller/push/push.go:127 +0xf7


```

--- a/sdk/metric/aggregator/ddsketch/ddsketch.go
+++ b/sdk/metric/aggregator/ddsketch/ddsketch.go
@@ -45,9 +45,10 @@ var _ aggregator.Distribution = &Aggregator{}
 // New returns a new DDSketch aggregator.
 func New(cfg *Config, desc *export.Descriptor) *Aggregator {
 	return &Aggregator{
-		cfg:     cfg,
-		kind:    desc.NumberKind(),
-		current: sdk.NewDDSketch(cfg),
+		cfg:        cfg,
+		kind:       desc.NumberKind(),
+		current:    sdk.NewDDSketch(cfg),
+		checkpoint: sdk.NewDDSketch(cfg),
 	}
 }
 

